alws/crud.py

```diff
@@ -23,7 +23,8 @@
 from alws.utils.distro_utils import create_empty_repo
 
 
-__all__ = ['create_build', 'get_builds', 'create_platform', 'get_platforms']
+__all__ = ['create_build', 'get_builds', 'create_platform',
```

Comment:
FYI. This decoration of the code is better than current because you can add/remove/comment each element with no additional efforts by an one hotkey)

```
lst = [
  'element1',
  'element2',
  'element3',
]
```